pkg/okteto/namespace.go

```diff
@@ -121,10 +121,12 @@ func (c *namespaceClient) Delete(ctx context.Context, namespace string) error {
 	var mutation struct {
 		Space struct {
 			Id graphql.String
-		} `graphql:"deleteSpace(id: $id)"`
+		} `graphql:"deleteSpace(id: $id, force: $force)"`
 	}
 	variables := map[string]interface{}{
 		"id": graphql.String(namespace),
+		// force is disabled to delete namespace using destroy-all-namespace job
```

Comment:
@ifbyol i was having doubts here, should we pass the value or should i prepare a param to be pass though from the command?

Reply:
Good question, we didn't consider to add the `--force` flag in the CLI. Leave it as it is for now to not block the PR. We can add a ticket later to include.

@rlamana Should we consider to add `--force` flag to this command? It wasn't considered orginally but maybe it make sense to create a ticket to include it at some point (without blocking this PR)

Reply:
It makes sense. There shouldn't be core operations like this that can be done from the UI, but not from the CLI. Create the ticket to tackle it.